File: js/q19.js
var arr=[1,2,3,4,5]
var count=0

for(i=0;i<arr.length-1;i++){
    if(arr[i]<arr[i+1]){
        count++
    }
}

if(count==arr.length-1){
    console.log("Array is in Ascending order.")
}else{
    console.log("Array is not in ascending order.")
}

File: clangua/q19.c
#include <stdio.h>
#include <string.h>

int main() {
    
    int arr[]={1,2,3,4,5};
    int i,count=0;
    int len=sizeof(arr)/sizeof(arr[0]);
    for(i=0;i<len-1;i++){
        if(arr[i]<arr[i+1]){
            count++;
        }
    }
    if(count==len-1){
        printf("Array is in ascending order.");
    }else{
        printf("Array is not in ascending order.");
    }

    return 0;
}